recipes/timers-and-such/direct/hparams/train_with_wav2vec.yaml

```diff
@@ -0,0 +1,199 @@
+# ############################################################################
+# Model: Direct SLU with WaV2VEC
+# Encoder: Pre-trained Wav2vec
+# Decoder: GRU + beamsearch
+# Tokens: BPE with unigram
+# losses: NLL
+# Training: Timers and Such
+# Authors:  Abdelmoumene Boumadane, AbdelWahab Heba, Loren Lugosch 2021
+# ############################################################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1986
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+experiment: wave2vec_classification_seq2seq
+output_folder: !ref results_SLU/<experiment>/<seed>
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# URL for the wav2vec2 model, you can change to benchmark diffrenet models
+wav2vec2_hub: "facebook/wav2vec2-base-960h"
+
+# Data files
+data_folder: !PLACEHOLDER # e.g, /localscratch/timers-and-such
+data_folder_rirs: !ref <data_folder>
+train_splits: ["train-real"]
+csv_train: !ref <output_folder>/train-type=direct.csv
+csv_dev_real: !ref <output_folder>/dev-real-type=direct.csv
+csv_dev_synth: !ref <output_folder>/dev-synth-type=direct.csv
+csv_test_real: !ref <output_folder>/test-real-type=direct.csv
+csv_test_synth: !ref <output_folder>/test-synth-type=direct.csv
+csv_all_real: !ref <output_folder>/all-real-type=direct.csv
+tokenizer_file: https://huggingface.co/speechbrain/slu-timers-and-such-direct-librispeech-asr/resolve/main/tokenizer.ckpt
+skip_prep: False
+ckpt_interval_minutes: 15 # save checkpoint every N min
+test_on_all_real: False
+
+
+# Training parameters
+number_of_epochs: 50
+batch_size: 4
+lr: 0.0004
+lr_wav2vec: 0.00001
+
+#freeze all wav2vec
+freeze_wav2vec: False
+
+#set to true to freeze the CONV part of the wav2vec model
+freeze_wav2vec_conv: True
```

Comment:
this maybe added into the wav2vec lobe.. !

Reply:
I did it on local i'll push on a pull request, so you can choose to freeze the features extractor when instantiating a wav2vec model which is more convenient